.github/workflows/configurations.yml

```diff
@@ -0,0 +1,158 @@
+name: Configurations
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+  workflow_dispatch: {}
+
+env:
+  DOCKER_USR: ${{ secrets.UPBOUND_ROBOT_USR }}
+
+jobs:
+  getting-started-with-aws:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Set tag
+        run: echo "::set-output name=VERSION_TAG::$(git describe --dirty --always --tags | sed 's/-/./2' | sed 's/-/./2' )"
```

Comment:
Nit: Could be worth adding a comment with an example of what this will produce; I had to run the command to find out.

Reply:
SGTM, I pulled this right from `build` logic so I'll add a link to that